backend/inspirehep/migrator/tasks.py

```diff
@@ -342,6 +349,9 @@ def migrate_record_from_mirror(
         return None
 
     if "$schema" in json_record:
+        if does_record_have_blacklisted_pid_type(json_record):
+            return
+
         ensure_valid_schema(json_record)
```

Comment:
I think you should move it after this line (otherwise you might not have a valid schema path yet)

Reply:
I was thinking why validate a record if we won't use it.

Reply:
it's a misnomer, `ensure_valid_schema transforms the `$schema` into a full URI instead of `jobs.json`.

Reply:
ahh it totally is, that's sad :(